c/aws-c-common/memcpy_using_uint64_harness.i

```diff
@@ -5930,6 +5931,9 @@ void memcpy_using_uint64_harness() {
     char s[160];
     char d1[160];
     char d2[160];
+    for (int i; i < 160; i++) {
```

Comment:
Missing `=0` initialiser.

Reply:
:facepalm: Of course! Thanks for noticing.

I wonder why the compiler did not flag this when I checked with `make`.

Reply:
Wow, I completely missed this!